scripts/README.md

```diff
@@ -17,7 +17,34 @@
   * Commit changes and push to origin
   * Open github to submit the PR
 
-## Update sitemap
+### Extracting texts from js code:
+Texts that used in js code would automatically be extracted during the translation update process (3rd step above: Updating the source file).
 
-* List of paths to include in `sitemap.xml` is here: [sitemap.js](sitemap.js)
+In order to make sure that the extractor code won't miss any string to push to translations, please:
+
+1. Refactor the code so that the first argument passed to `localize()` method, be a string literal.
+    i.e.
+    ```js
+    const text = localize(is_started ? 'Sell at market' : 'Sell');
+    ```
+    could change to:
+    ```js
+    const text = is_started ? localize('Sell at market') : localize('Sell');
+    ```
+2. If there is no way to have the string literal in js code (i.e. API texts which are not translated), add them to `scripts/js_texts/static_strings_app.js`.
+3. At the end, when you're sure that a strings already been taken care of somewhere else, just put this comment `/* localize-ignore */` right after the first argument of `localize()` call to ignore it from tests.
+
+### `js_texts` folder contents:
+```
+js_texts:
+    ├── extracted_strings_app.js
+    └── static_strings_app.js
+```
+- `extracted_strings_app.js`: Contains extracted strings. It's for debugging purpose only and shouldn't change manually.
```

Comment:
I think `shouldn't be updated manually` or `shouldn't be changed manually` would be clearer